src/Swarm/Game/Robot.hs

```diff
@@ -69,6 +69,7 @@ module Swarm.Game.Robot (
   runningAtomic,
 
   -- ** Creation & instantiation
+  emptyRobotContext,
```

Comment:
I think this should be listed in the "Robot context" section earlier in the export list.